packages/icons/icons.json

```
@@ -2886,168 +2886,168 @@
         "group": "data",
         "codepoint": 62125
     },
     {
         "displayName": "Series: derived",
         "iconName": "series-derived",
-        "tags": "timeseries, plot, line, data, chart, transform, transformed, compute, computed, modify, modified, alter, altered, adjust, adjusted",
+        "tags": "timeseries, plot, line, data, chart, transform, transformed, compute, computed, modify, modified, altered, adjusted, trend, forecast, projection, prediction, expectation",
         "group": "data",
         "codepoint": 62124
     },
     {
         "displayName": "Series: configuration",
         "iconName": "series-configuration",
-        "tags": " timeseries, plot, line, data, chart, edit, modify, customize, adjust, alter, transform",
+        "tags": "timeseries, plot, line, data, chart, edit, modify, customize, adjust, alter, transform, trendline, goal, mountain sun, settings, configure",
         "group": "data",
         "codepoint": 62123
     },
     {
         "displayName": "Console",
         "iconName": "console",
-        "tags": "terminal, coding",
+        "tags": "terminal, coding, code block, command line, input",
         "group": "interface",
         "codepoint": 61797
     },
     {
         "displayName": "Compass",
         "iconName": "compass",
-        "tags": "map, direction",
+        "tags": "map, direction, north south east west, magnetic poles, navigation",
         "group": "interface",
         "codepoint": 61794
     },
     {
         "displayName": "Walk",
         "iconName": "walk",
-        "tags": "transportation, run, move",
+        "tags": "transportation, run, move, path, marching, going, person walking, march",
         "group": "interface",
         "codepoint": 62228
     },
     {
         "displayName": "Taxi",
         "iconName": "taxi",
-        "tags": "transportation, car, move",
+        "tags": "transportation, car, move, police vehicle",
         "group": "interface",
         "codepoint": 62179
     },
     {
         "displayName": "Train",
         "iconName": "train",
-        "tags": "transportation, public, move",
+        "tags": "transportation, public, move, rail, subway",
         "group": "interface",
         "codepoint": 62200
     },
     {
         "displayName": "Heart: broken",
         "iconName": "heart-broken",
-        "tags": "love, like, organ, human, feelings, split, torn, failed, health",
+        "tags": "love, like, organ, human, feelings, split, torn, failed, health, heartbreak, sadness, heart attack, unhealthy",
         "group": "miscellaneous",
         "codepoint": 61927
     },
     {
         "displayName": "Join: inner",
         "iconName": "inner-join",
-        "tags": "circles, combine, connect, add, part, slice",
+        "tags": "circles, combine, connect, add, part, slice, eye, cell, intersection, overlap, option, navigate, vision, venn diagram",
```

we already have a good "eye" icon so I don't think we need to conflate this with that one.

```suggestion
        "tags": "circles, combine, connect, add, part, slice, cell, intersection, overlap, venn diagram",
```